src/components/MVP/CovidSiteAlert/CovidSiteAlert.tsx

```diff
@@ -7,6 +7,9 @@ import styles from './CovidSiteAlert.module.scss'
 const CovidSiteAlert = () => {
   return (
     <SiteAlert
+      //#TODO: Best way to check that a component rendered
+      // on a page re: testing? This seems easiest.
```

Comment:
This is a good reference for best ways to select components: https://testing-library.com/docs/queries/about#priority

_also_ if you use `getByRole()` in a test but there's no result, it will helpfully return the list of roles that are present on the page! that is usually how I will figure out if there's a role I can use if I'm not sure. so I tried that in the test and found this result:
```
Here are the accessible roles:

      region:

      Name "Site alert":
      <section
        aria-label="Site alert"
        class="usa-site-alert usa-site-alert--info usa-site-alert--no-heading usa-site-alert--no-icon covidSiteAlert"
        data-testid="covid-alert"
      />
```

so in this case I think it can use `screen.getByRole('region', { name: 'Site alert' })` 🎉